config/plugins/barefoot.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+import click
+import json
+import subprocess
+from sonic_py_common import device_info
+
+def abort_if_false(ctx, param, value):
+    if not value:
+        ctx.abort()
+
+@click.group()
+def barefoot():
+    pass
+
+@barefoot.command()
+@click.option('-y', '--yes', is_flag=True, callback=abort_if_false,
```

Comment:
>yes [](http://example.com/codeflow?start=23&length=3)

How do you use this option?

Reply:
The `--yes` option guards the user from unwanted restart of `swss` service.
If not specified, the command will prompt the user with the following message:
`Swss service will be restarted, continue? [y/N]:`
Answering `n` will lead the command to abort.

Reply:
I don't see you reference the option in code. I am wondering if it works.

Reply:
`callback=abort_if_false` specifies that function `abort_if_false` will be invoked with `--yes` parameter value after the parameter is handled and before `profile` function is called.
`abort_if_false` function aborts the command if `--yes` parameter value is `false`.
`prompt='Swss service will be restarted, continue?'` asks for user input if `--yes` parameter is not provided from the command line.
Please see Click documentation for an example:
https://click.palletsprojects.com/en/8.0.x/options/#yes-parameters